lib/urcli-money.js

```diff
@@ -132,15 +139,22 @@ class Report {
     console.log(chalk.blue(`Earnings Report for ${startDate} to ${endDate}:`))
 
     for (let p in this.projects) {
-      let {name, id, ungradeable, passed, failed, earned} = this.projects[p]
+      let {name, id, ungradeable, passed, failed, earned, hours} = 
+          this.projects[p]
       console.log(`
       ${chalk.white(`Project: ${name} (${id}):`)}
           ${chalk.white(`Total reviewed: ${passed + failed}`)}
           ${chalk.white(`Ungradeable: ${ungradeable}`)}
           ${chalk.white(`Earned: ${earned}`)}
+          ${chalk.white(`Working Hours: ${hours.toFixed(2)}`)}
+          ${chalk.white(`Earned per Hour: ${(earned/hours).toFixed(2)}`)}
```

Comment:
And perhaps this could be labeled differently as well to reflect that it's not accurate.